src/test/java/org/checkerframework/specimin/GlobalVariables.java

```diff
@@ -0,0 +1,19 @@
+package org.checkerframework.specimin;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * This test checks that a simple Java file with no dependencies whatsoever and a single target
+ * method is returned unaltered by specimin.
```

Comment:
For each new test, this comment should be modified to explain the test's purpose.